docker_daemon/check.py

```diff
@@ -183,11 +183,11 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         self._service_discovery = agentConfig.get('service_discovery') and \
             agentConfig.get('service_discovery_backend') == 'docker'
 
-        self.collect_labels_as_tags = agentConfig.get('docker_labels_as_tags', ','.join(DEFAULT_LABELS_AS_TAGS))
-        if self.collect_labels_as_tags:
-            self.collect_labels_as_tags = [label.strip() for label in self.collect_labels_as_tags.split(',')]
+        self.global_labels_as_tags = agentConfig.get('docker_labels_as_tags')
```

Comment:
you don't need to set a member variable (`self.`) here, as it's not reused outside that method, just set a local variable so it's removed from memory